template_app/package.json

```diff
@@ -17,8 +17,6 @@
     "test:integration": "playwright test",
     "test:unit": "vitest",
     "test:unit-run": "vitest run",
-    "test:integration": "playwright test",
```

Comment:
Intentional diff or rebase conflict?

Reply:
Not intentional, so sounds like rebase conflict!

Reply:
Looking at this again, I think these have been automatically removed as duplicates of L17-L18